examples/bindeps/src/main.rs

```diff
@@ -0,0 +1,8 @@
+fn main() {
+    println!(env!("CARGO_CDYLIB_FILE_FOO"));
```

Comment:
We should include a `BIN` dependency here too to make sure it is wired up properly as well
```suggestion
    println!(env!("CARGO_CDYLIB_FILE_FOO"));
    println!(env!("CARGO_BIN_FILE_FOO"));
```